glog.go

```diff
@@ -702,7 +702,10 @@ func (l *loggingT) outputWithDepth(s severity, buf *buffer, extraDepth int, deta
 	data := buf.Bytes()
 	n, _ := output.Write(data)
 	if s == fatalLog {
-		output.Write(stacks(false))
+		// If we got here via Exit rather than Fatal, print no stacks.
+		if atomic.LoadUint32(&fatalNoStacks) == 0 {
+			output.Write(stacks(false))
```

Comment:
Why not adopt the upstream change? Basically lines 709–714: https://github.com/golang/glog/blob/23def4e6c14b4da8ac2ed8007337bc5eb5007998/glog.go#L709-L714.

Note that the current code also doesn't do the 1 exit code (instead of 255) that is suggested by the godoc.

Reply:
Makes sense, changed to use status code 1 on exit, 255 on fatal